sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/optimizer/InjectRuntimeFilter.scala

```diff
@@ -29,7 +29,11 @@ import org.apache.spark.sql.internal.SQLConf
 import org.apache.spark.sql.types._
 
 /**
- * Insert a filter on one side of the join if the other side has a selective predicate.
+ * Insert a runtime filter on one side of the join (we call this side the application side)
+ * if we can extract a plan (we call this plan the creation side) from the other side and
```

Comment:
it's weird to call an extracted plan "side". A join has two sides, and we extract a plan from the creation side to build the runtime filter.

Reply:
Sounds reasonable.